### ISSUE TITLE
Clear should check if the element is editable

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#208]
**Expected behavior**
Clear should check if the element is editable.

**Actual behavior**
It executes on non-editable elements and also the action is marked as successful

**Steps to replicate**
* Run the following script
```
const { openBrowser, goto, clear, write, press, button } = require('taiko');

(async () => {
    try {
        await openBrowser();
        await goto("makemytrip.com");
        await clear(button({"id":"searchBtn"}));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
> Here button cannot be cleared however the action is successful

**Version**
```
commit - 0d5d17cc301cdf3d1c7700a08a996af63a0d4731
```
Related to #65

**Comment by NivedhaSenthil** [comment]
Fix available in bfc612a

**Comment by NivedhaSenthil** [comment]
Fix available in bfc612a

**Comment by NivedhaSenthil** [comment]
Fix available in bfc612a
